lib/opensource/Node-SMB/README.md

```diff
@@ -0,0 +1,78 @@
+# node-ntlm, NTLM authentication and Samba LM/NT hash library
```

Comment:
Can we cut this? Maybe replace it with something similar to https://github.com/microsoft/typed-rest-client/blob/master/lib/opensource/node-http-ntlm/readme.txt